com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -177,6 +192,25 @@ public void ConfigureInput(ScriptableRenderPassInput passInput)
             m_Input = passInput;
         }
 
+        internal void ConfigureColorStoreAction(RenderBufferStoreAction storeAction, uint attachmentIndex = 0)
+        {
+            m_ColorStoreActions[attachmentIndex] = storeAction;
+        }
+
+        internal void ConfigureColorStoreActions(RenderBufferStoreAction[] storeActions)
+        {
+            int count = Math.Min(storeActions.Length, m_ColorStoreActions.Length);
+            for (uint i = 0; i < count; ++i)
+            {
+                m_ColorStoreActions[i] = storeActions[i];
+            }
+        }
+
+        internal void ConfigureDepthStoreAction(RenderBufferStoreAction storeAction)
+        {
+            m_DepthStoreAction = storeAction;
+        }
+
```

Comment:
these 3 methods are public in master, was it intentional to change them to internal?

Reply:
also, when they are changed back. to public, we will need to add API docs (which I forgot to add in the original PR, ooops).
I've added the docs in this [PR](https://github.com/Unity-Technologies/Graphics/pull/5221), currently waiting for review, so once my docs PR is approved we should adds the docs too to the backport